org.flatpak.popout3d.yml

```diff
@@ -55,8 +55,8 @@ modules:
       cxxflags: "-std=c++0x"
     sources:
       - type: archive
-        url: https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.4/wxWidgets-3.0.4.tar.bz2
-        sha256: 96157f988d261b7368e5340afa1a0cad943768f35929c22841f62c25b17bf7f0
+        url: https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.5.1/wxWidgets-3.0.5.1.tar.bz2
+        sha256: 406ac736f61d88a3a866aa501e01e408a642c6e7
```

Comment:
This is not a sha256, it is too short.

(the build will fail)

Example:
https://github.com/flathub/net.sourceforge.Hugin/blob/master/net.sourceforge.Hugin.json#L54

Reply:
That was what was on the webpage, I've replaced it with the one the error message said it was looking for.

Reply:
They are sha1 not sha256